website/content/docs/concepts/oidc-provider.mdx

```diff
@@ -0,0 +1,144 @@
+
+The openid scope is a unique case scope that may not be modified or deleted. The scope will exist in Vault and is supported by each provider by default. The scope represents the minimum set of claims required by the OIDC specification for inclusion in ID tokens. As such, templates may not contain top-level keys that overwrite the claims populated by the openid scope.
+
+The following defines the claims key and value mapping for the openid scope:
+
+* `iss`- configured issuer of the provider
+* `sub`- unique entity ID of the Vault user
+* `aud`- iD of the client
+* `iat`- time of token issue
+* `exp`- time of token issue + ID token TTL
+
+### Client registration
+
+A client resource allows the relying party by providing metadata about itself to Vault to [dynamically register](https://openid.net/specs/openid-connect-registration-1_0.html).
+
+The client must have the following configuration parameters:
+
+* **Redirect URIs**: limits the valid redirect URIs in an authentication request
+* **Assignments**: determines who can authenticate with the client
+* **Key**: signing of the ID tokens
+*	**ID token TTL**: specifies the time-to-live for ID tokens
+* **Access token TTL**: establishes the time-to-live for access tokens
+
+A `client_id` and `client_secret` are generated and returned after a successful client registration. Their values are strings using the base62 character set. The `client_id` will have 32 characters, and the `client_secret` will have a prefix of `hvo_secret`. The `client_id`  uniquely identifies the client. The `client_secret` will be used to authenticate to the token endpoint as described in [client authentication](https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication).
+
+The `key` parameter is required. The user must create a `key` as a required parameter of the client configuration.
+
+~> **Note**: At least one of the redirect URIs of a client must exactly match the `redirect_uri` parameter used in an authentication request initiated by the client.
+
+### Assignments
+
+Assignment resources are referenced by clients via the `assignments` parameter. This parameter limits the set of Vault users allowed to authenticate. The assignments of an associated client are validated during the authentication request, ensuring that the Vault identity associated with the request is a member of the assignment's entities or groups.
+
+### Keys
+
+Key resources are referenced by clients via the key parameter. This parameter specifies the key that will be used to sign ID tokens for the client. The key resource established is used to support identity tokens. See existing [documentation](/api-docs/secret/identity/tokens#create-a-named-key) for details on keyring management, supported signing algorithms, rotation periods, and verification TTLs. Currently, a key referenced by a client cannot be changed.
```

Comment:
```suggestion
Key resources are referenced by clients via the key parameter. This parameter specifies the key that will be used to sign ID tokens for the client. See existing [documentation](/api-docs/secret/identity/tokens#create-a-named-key) for details on keyring management, supported signing algorithms, rotation periods, and verification TTLs. Currently, a key referenced by a client cannot be changed.
```

Reply:
I feel like maybe it is clearer without that sentence. Although maybe we do want to clarify that the `key` resource/api used by the Vault OIDC Provider feature is the same `key` resource/api used by the identity token system.